app/models/journeyDomain/item/packages/PackageDomain.scala

```diff
@@ -34,17 +34,26 @@ case class PackageDomain(
 
   override def toString: String = `type`.toString
 
+  def asString(implicit messages: Messages): String = PackageDomain.asString(`type`, numberOfPackages)
+
   override def routeIfCompleted(userAnswers: UserAnswers, mode: Mode, stage: Stage): Option[Call] = Some {
     stage match {
       case AccessingJourney =>
         controllers.item.packages.index.routes.PackageTypeController.onPageLoad(userAnswers.lrn, mode, itemIndex, packageIndex)
-      case CompletingJourney => Call(GET, "#")
+      case CompletingJourney => controllers.item.packages.routes.AddAnotherPackageController.onPageLoad(userAnswers.lrn, mode, itemIndex)
     }
   }
 }
 
 object PackageDomain {
 
+  def asString(packageType: PackageType, numberOfPackages: Option[Int])(implicit messages: Messages): String =
+    numberOfPackages.fold(
+      messages("packages.value.withoutNumberOfPackages", packageType.toString)
+    )(
+      quantity => messages("packages.value.withNumberOfPackages", quantity, packageType.toString)
+    )
```

Comment:
Perhaps easier to do s"${numberOfPackages.getOrElse(1)} packageType"? Don't mind this approach though